docs/docs/ports.md

```diff
@@ -42,6 +42,26 @@ If you are running Marathon within a [DC/OS cluster](https://dcos.io/get-started
 
 *servicePort*: When you create a new application in Marathon (either through the REST API or the front end), you may assign one or more service ports to it. You can specify all valid port numbers as service ports or you can use 0 to indicate that Marathon should allocate free service ports to the app automatically. If you do choose your own service port, you have to ensure yourself that it is unique across all of your applications.
 
+## Use Cases for Different ports
+
+The 3 ports defined (containerPort, hostPort, servicePort) along with the label for a VIP_0 in a port mapping can be confusing.  The following is another way of considering these ports using an ACME app as an example.  Lets say the actual application opens port :8080.   All applications on Mesos are hosted in containers. Containers can be run in bridge host mode.  When in host mode, the container will have the host network interface defined in the container namespace.  The container will not do any port magic and the application will open up port :8080 on the host network interface.   In this case there is no container port, and the `hostPort` is :8080.
+The container could be run in bridge mode.  In this case, the container gets it's own network interface and the `containerPort` is :8080.   In bridge mode, the container will open a pseudo random port on the host and bridge/NAT communication to the container port.   For example, lets say the port is :31000.   Now the `containerPort` is :8080, but the `hostPort` is 31000.  Clients to this service will open port 31000 on the host the service is running on and the container will route that to the internal port :8080 in the container.
```

Comment:
`it's` -> `its`. Should we be more consistent w.r.t a colon prepended to ports: `31000` vs `:31000`?